subsys/bootloader/bootloader.h

```diff
@@ -0,0 +1,72 @@
+/*TODO: Remove when multi-image has been merged*/
+#ifndef BOOTLOADER_H__
+#define BOOTLOADER_H__
+
+#if CONFIG_SOC_NRF52840
+/* partition@0 */
+#define DT_FLASH_AREA_SECURE_BOOT_OFFSET	0
+#define DT_FLASH_AREA_SECURE_BOOT_SIZE	16384
+
+/* partition@10000 */
+#define DT_FLASH_AREA_S1_OFFSET	65536
+#define DT_FLASH_AREA_S1_SIZE		49152
+
+/* partition@1C000 */
+#define DT_FLASH_AREA_APP_OFFSET	114688
+#define DT_FLASH_AREA_APP_SIZE	929792
+
+/* partition@4000 */
+#define DT_FLASH_AREA_S0_OFFSET		16384
+#define DT_FLASH_AREA_S0_SIZE		49152
+
+/* partition@4000 */
+#define DT_FLASH_AREA_S0_OFFSET	16384
+#define DT_FLASH_AREA_S0_SIZE		49152
+
+/* partition@FF000 */
+#define DT_FLASH_AREA_PROVISION_OFFSET	1044480
```

Comment:
You could do `FLASH_SIZE-PROVISION_SIZE`

Reply:
Yes, that's what I was thinking. That should be sufficient to support all nRF5x devices.

Reply:
Yeah it works for the nrf5 devices. Couldn't find a general config variable for it so it will have to be
```
#if CONFIG_SOC_NRF52840  ||\
    CONFIG_SOC_NRF52832  ||\
    CONFIG_SOC_NRF52810  ||\
    CONFIG_NRF51822_QFAA ||\
    CONFIG_NRF51822_QFAB ||\
    CONFIG_NRF51822_QFAC
```

Reply:
Done